Time/DayAndNightClock

```diff
@@ -0,0 +1,85 @@
+#!/bin/bash
+
+# <bitbar.title>DayAndNightClock</bitbar.title>
+# <bitbar.version>v1.0</bitbar.version>
+# <bitbar.author>FusionX</bitbar.author>
+# <bitbar.author.github>FusionX13</bitbar.author.github>
+# <bitbar.desc>A clock that can do many things.</bitbar.desc>
+# <bitbar.image>https://ibb.co/CBLByKf</bitbar.image>
+# <bitbar.dependencies>bash</bitbar.dependencies>
+# <bitbar.abouturl>http://url-to-about.com/</bitbar.abouturl>
+
+# Gets Emoji Moon
+get_phase_day () {
+  local lp=2551443
+  local now=$(date -ju +"%s")
```

Comment:
```
In /Users/bhartvig/Projects/bitbar-plugins/Time/DayAndNightClock.1h.sh line 15:
  local now=$(date -ju +"%s")
```
        ^-^ SC2155: Declare and assign separately to avoid masking return values.